src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -434,7 +428,15 @@ public void before() throws Throwable {
      * Configures the update center setting for the test.
      * By default, we load updates from local proxy to avoid network traffic as much as possible.
      */
-    protected void configureUpdateCenter() throws Exception {
```

Comment:
Protected methods can be overridden in other libraries. This change breaks binary compatibility from what I see

Reply:
This method did exist but was not doing anything. Thanks for letting me know!

Reply:
AFAICT it was not fixed, right?
Usage: https://github.com/jenkinsci/jenkins-test-harness/blob/e72678d347373228b6d81097d792e968376a6aa1/src/main/java/org/jvnet/hudson/test/HudsonTestCase.java#L337